### DISCUSSION TITLE
Issue with new open core legacy patcher 1.1 version latest available in Mac OS Sonoma 14

### DISCUSSION THREAD

**Discussion by OmeshGithub** [community/community#73073] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Having iMac with intel chip 2.7GHz Quad Core and NVIDIA GeForce GT 640M 512 MB 8GB RAM when I load the open core 1.1 version the system becomes unstable and screen starts to flash going blank and some windows getting cropped and becomes unresponsive. The system seems working OK with Sonoma 14 with old legacy patcher, Can you please help

**Comment by tribal666** [comment]
Hi everybody,

Same probs with NVIDIA GeForce GT 640M 512 MB. Do you find a solution? Many thks.
